app/assets/javascripts/questions.coffee

```diff
@@ -1,3 +1,11 @@
 # Place all the behaviors and hooks related to the matching controller here.
 # All this logic will automatically be available in application.js.
 # You can use CoffeeScript in this file: http://coffeescript.org/
+
+$(document).ready ->
+  $('input:radio').click ->
```

Comment:
I mean that this code will affect all application
be specific for input (add class)
because with current code - each click on every radio input will  add active to some parent class (btw if possible it's better avoid using things like .parent() )